africanus/calibration/tests/test_phase_only.py

```diff
@@ -56,10 +56,10 @@ def make_dual_pol_data(sigma_n, n_dir, sigma_f):
     # simulate model data (pure Stokes I)
     for dir in range(n_dir):
         this_lm = lm[dir].reshape(1, 2)
-        this_flux = np.tile(flux[dir], (n_chan)).reshape(1, n_chan)
+        this_flux = np.tile(flux[dir], (n_chan,
+                            n_cor)).reshape(1, n_chan, n_cor)
```

Comment:
I fear these ``this_naming_convention`` will open a portal to some hellish coding realm but perhaps it will be OK :->

Reply:
Old habits... Think we will survive